changelogs/SDK.CHANGELOG.2024.md

```diff
@@ -1,7 +1,40 @@
+	* The `DetermineServiceURL` method on the `ClientConfig`, the base class of service client configs like `AmazonS3Config`, was removed. This had been marked obsolete and was tied to an internal endpoint resolution system that was removed in V4. Application code calling this method should switch to using the `DetermineServiceOperationEndpoint` method on the service client.
+	* When using credential providers that rely on Amazon Security Token Service (STS) the calls will always use the regional endpoint. V3 by default used `us-east-1` regardless of the configured region when running in the public partition. As part of this change the `StsRegionalEndpointsValue` enum was removed. Any code using that enum should be removed.
+	* On service client configs the `RetryMode` defaults to `Standard` instead of `Legacy`. The `Legacy` enum value was removed.
+	* On service client configs the `DefaultConfigurationMode` defaults to `Standard` instead of `Legacy`. The `Legacy` enum value was removed.
+	* The System.Buffers, System.Memory and System.Text.Json packages have been added as dependencies for the .NET Framework and .NET Standard targets.
+	* The embedded `endpoints.json` file in AWSSDK.Core was removed. This was a 1 MB json file that was parsed at startup for the SDK to determine the service endpoint for a region. Endpoint resolution has been replaced with a new system that generates rules into each individual service assembly for determining the service endpoints.
+	* Community [PR #3359](https://github.com/aws/aws-sdk-net/pull/3359) Optimize GetExtension execution time on .NET 8. Thanks [Steven Weerdenburg](https://github.com/stevenaw)
+    * Community [PR #3293](https://github.com/aws/aws-sdk-net/pull/3293) improving `AWSSDK.ToHex` performance. Thanks [Steven Weerdenburg](https://github.com/stevenaw)
+    * Community [PR #3292](https://github.com/aws/aws-sdk-net/pull/3292) improving `AWSSDK.CopyTo` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3324](https://github.com/aws/aws-sdk-net/pull/3324) optimizing the AWS SigV4 signer. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3363](https://github.com/aws/aws-sdk-net/pull/3363) improving `AWSSDKUtils.ParameterAsString` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3365](https://github.com/aws/aws-sdk-net/pull/3365) improving `AWSSDKUtils.DetermineService` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3307](https://github.com/aws/aws-sdk-net/pull/3307) Optimizing `AWSSDKUtils.UrlEncode` performance. Thanks [Daniel Marbach](https://github.com/danielmarbach)
+    * Community [PR #3425](https://github.com/aws/aws-sdk-net/pull/3425) Avoid allocating byte[] when converting MemoryStream to String. Thanks [Paulo Morgado](https://github.com/paulomorgado)
+
```

Comment:
These extra lines (35-37) aren't needed, right? I don't see them in the `CHANGELOG.ALL` file.

Reply:
Removed